### ISSUE TITLE
Jackson deserialization fails for NotificationRequest

### ISSUE THREAD

**Issue by deepu105** [Adyen/adyen-java-api-library#476]
**Describe the bug**
Jackson deserialization fails for `class com.adyen.model.notification.NotificationRequest` with below error

```
MappingJackson2HttpMessageConverter : Failed to evaluate Jackson deserialization for type 
[[simple type, class com.adyen.model.notification.NotificationRequest]]: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: 
Conflicting getter definitions for property "notificationItems": 
com.adyen.model.notification.NotificationRequest#getNotificationItems(0 params) vs 
com.adyen.model.notification.NotificationRequest#getNotificationItemContainers(0 params)
```

Not sure if same issue is present fro Gson as well

**How to reproduce**
Try to deserialize  notification JSON to `NotificationRequest`

**Expected behavior**
Proper deserialization without issues

**Desktop (please complete the following information):**
 - Java Version: 8
 - Library Version: 11.0.1

**Additional context**

Solution could be to refactor conflicting getters or to specify getter to use in Jackson annotation

**Comment by Aleffio** [comment]
Hi @deepu105 ,

thanks for raising the issue,
we created an internal ticket for this.

Regards,
Alessio
Adyen

**Comment by saquibsayyad** [comment]
Hi @deepu105,

This issue has been fixed now in #480. Let us know if you still face again.

Thanks,
Saquib
